docker-compose.yml

```diff
@@ -0,0 +1,53 @@
+version: '2.1'
+
+services:
+  postgres:
+    user: postgres
+    image: postgres
+    ports: 
+      - "5432:5432"
+    environment:
+      POSTGRES_USER: 'postgres'
+      POSTGRES_DB: 'elasticapm_test'
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready"]
+      interval: 30s
```

Comment:
Should we make this much more frequent like `1s`? I'm assuming this would make the tests starting quicker as the container is recognised as healthy earlier.